Added arguments to main method (#94)
baijum/selenium-python: source/getting-started.rst

The above result shows that the test has been successfully completed.

> Note: To run the above test in IPython or Jupyter, we need to pass arguments in main method as shown below:      

Reviewer: ```suggestion
Note: To run the above test in IPython or Jupyter, you should pass a couple of arguments to the `main` function as shown below:
```
